### ISSUE TITLE
Adding Kernal ROM results in glitches in the samples in MUSICIANS\E\Encore\Spread_Eagle_Idaho.sid

### ISSUE THREAD

**Issue by acrouzet** [libsidplayfp/sidplayfp#55]
I'm using the V3 Kernal. Taking the Kernal ROM out of sidplayfp's search paths fixes the glitches. I'm unsure if this is a problem with the Kernal itself, or if its an emulation issue, or something else.

**Comment by ruby-R53** [comment]
i can reproduce it, so probably not the former

i'd guess it's an emulation issue because that's a similar case to #39, where removing the kernal rom fixed the issue as well (although it had more to do with audio buffer stuff, still related tho')

**Comment by drfiemost** [comment]
More likely a bad rip.
Did you try with other players, possibly vsid?

**Comment by acrouzet** [comment]
It works in jsidplay2.

**Comment by drfiemost** [comment]
What kind of glitches? I can't hear anything wrong here.

**Comment by ruby-R53** [comment]
i can't either anymore, i tried playing it using the reSID engine once and it sounded fine there, then i went back to reSIDfp and it somehow played normally too

what i heard when i was able to reproduce however seemed to be the samples being played way too louder than what they were supposed to be

**Comment by acrouzet** [comment]
Yeah, I can't recreate it anymore either. Strange.

**Comment by drfiemost** [comment]
So it's a random behaviour... is this on Windows?
Please add some details if you can reproduce this consistently in the future, it may be some uninitialized var or something.

**Comment by ruby-R53** [comment]
> So it's a random behaviour...

yeah i'm still able to reproduce it and it plays correctly only after a bunch of restarts

it might play correctly at launch if i'm lucky enough too
